### PR TITLE
docs(guide/Conceptual Overview): fix external api example

### PR DESCRIPTION
**Pull request by Narretz**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
docs

**What is the current behavior? (You can also link to an open issue here)**
Example is broken in the snapshot because of changes to JSONP

**Other information**:

The example will still be broken when opened in a plnkr, because they use the stable version instead of the snapshot.

### CODE REVIEW

File: docs/content/guide/concepts.ngdoc [hunk 1]
```diff
@@ -318,13 +318,21 @@ The following example shows how this is done with Angular:
...
+          'https://query.yahooapis.com/v1/public/**'
+        ]);
       }]);
   </file>
   <file name="finance3.js">
     angular.module('finance3', [])
       .factory('currencyConverter', ['$http', function($http) {
         var YAHOO_FINANCE_URL_PATTERN =
-              '//query.yahooapis.com/v1/public/yql?q=select * from ' +
+              'https://query.yahooapis.com/v1/public/yql?q=select * from ' +
```
Comment (gkalpak) [inline docs/content/guide/concepts.ngdoc]: Is this necessary? This won't work if the plnkr is loaded ovr `http:` iirc.

**Review by gkalpak — COMMENTED** [review]
BTW, I think we could just use `.get()` instead of `.jsonp()`.

**Comment by Narretz** [comment]
True, the api sets `Access-Control-Allow-Origin:"*"`. I'll make that change.

File: docs/content/guide/concepts.ngdoc (hunk 1 above)
Comment (Narretz) [inline docs/content/guide/concepts.ngdoc]: The plnkrs are loaded over https. I had to add it, otherwise it didn't match.

**Comment by gkalpak** [comment]
LGTM

**Pull request merged**